diff --git a/device-modules/drivers/misc/mediatek/imgsensor/src/isp3_m/Makefile b/device-modules/drivers/misc/mediatek/imgsensor/src/isp3_m/Makefile
--- a/device-modules/drivers/misc/mediatek/imgsensor/src/isp3_m/Makefile
+++ b/device-modules/drivers/misc/mediatek/imgsensor/src/isp3_m/Makefile
@@ -40,7 +40,13 @@ subdir-ccflags-y += \
 	-I$(IMGSENSOR_DRIVER_PATH)/$(MTK_PLATFORM)/seninf/ \
 	-I$(IMGSENSOR_DRIVER_PATH)/isp6s/seninf/ \
 	-I$(DEVICE_MODULES_PATH)/drivers/i2c/busses/ \
-	-DNO_I2C_MTK -DNO_CLK_METER -DIMGSENSOR_USE_RPM -DIMGSENSOR_DFS_CTRL_ENABLE -DSENINF_USE_WAKE_LOCK
+	-DNO_I2C_MTK -DNO_CLK_METER -DIMGSENSOR_USE_RPM -DSENINF_USE_WAKE_LOCK
+
+ifeq ($(COMMON_VERSION),v1_1)
+subdir-ccflags-y += -DDFS_CTRL_BY_OPP
+else
+subdir-ccflags-y += -DIMGSENSOR_DFS_CTRL_ENABLE
+endif
 
 define add_subdrv
 $(eval subdrv-rpath = $1)
